package.json

```diff
@@ -175,14 +175,38 @@
   "nyc": {
     "extends": "@istanbuljs/nyc-config-typescript",
     "extension": [
-      ".tsx"
+      ".tsx",
+      ".ts"
     ],
     "include": [
-      "src/**/*.tsx"
+      "src/**/*.tsx",
+      "src/**/*.ts"
     ],
     "exclude": [
-      "assets/",
-      "test/"
+      "src/__tests__",
+      "src/hooks/__mocks__",
+      "src/utils/__mocks__",
+      "src/utils/CustomError.ts",
+      " src/feature/controls_panel/",
+      " src/feature/results_panel/",
+      "src/feature/flow_chart_panel/components/nodes/PieChart.tsx",
```

Comment:
Ideally we want this exclude list to be small. Please go grep through this list and confirm that none of the files/classes here are being used in prod. If there's merged (non-test) code that's not being used in in prod that's not great either, so we should file an issue to follow up later.

If you wind up un-commenting some stuff and the coverage falls below 80% that's ok. We just need to make a follow up list so we know what tests to write in the future.